neural_compressor/adaptor/torch_utils/model_wrapper.py

```diff
@@ -329,6 +326,10 @@ def __init__(
             self.g_idx = None
 
     def pack(self, int_weight, scale, zp, bias, g_idx=None):
+        if self.use_optimum_format:
+            self.scales = self.scales.T
```

Comment:
Using in-place mode here, is it right?

Reply:
`Transpose` in Torch won't introduce a new tensor.